src/GraphQL/Resolver.hs

```diff
@@ -217,24 +217,11 @@ instance forall m ksN enum. (Applicative m, API.GraphQLEnum enum) => HasResolver
   type Handler m (API.Enum ksN enum) = enum
   resolve handler _ = (pure . ok . GValue.ValueEnum . API.enumToValue) handler
 
-
--- TODO: variables should error, they should have been resolved already.
---
--- TODO: Objects. Maybe implement some Generic object reader? I.e. if I do
--- data Greet = Greet { name :: Text, score :: Int } deriving Generic
--- then "instance ReadValue Greet" would fall back on a default reader that
--- expects Objects?
--- Maybe we can use advanced fallbacks like these:
--- https://wiki.haskell.org/GHC/AdvancedOverlap
-
-
--- Iterate through handlers (zipped together with their type
```

Comment:
this is more of a comment about what we do, but maybe I'll write that up in some other form later